speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -189,7 +189,7 @@ static void SetMaxTime(void*)
 /// @brief Handle input in the UserID textbox
 static void SetUserID(void*)
 {
-    char* m_userID = GfuiEditboxGetString(s_scrHandle, m_userIDId);
+    m_userID = GfuiEditboxGetString(s_scrHandle, m_userIDId);
     GfuiEditboxSetString(s_scrHandle, m_userIDId, m_userID);
```

Comment:
Don't capitalize abbreviation ID and I would personally give the identifiers referencing a GUI control a clearer name than just 'id'. For example: userIdControl
```suggestion
    m_userId = GfuiEditboxGetString(s_scrHandle, m_userIdControl);
    GfuiEditboxSetString(s_scrHandle, m_userIdControl, m_userId);
```